server/boot/adminUser.js

```diff
@@ -3,21 +3,23 @@
  */
 const { app } = require("../config/config");
 const { make } = require('../services/jwtService');
+const crypto = require('crypto');
 
 module.exports = App => {
   const { user } = App.models;
   user.create(
     {
       name: app.admin.name,
       email: app.admin.email,
+      password: crypto.randomBytes(20).toString('hex'),
```

Comment:
Let's change this as a security measure, we want to use `'base64'` since it's harder to guess.

Add a util function called generateRandomValueBase64. This will take a length value as param.
```javascript
var crypto = require('crypto')

function generateRandomValueBase64(length) {
  return crypto
    .randomBytes(Math.ceil((length * 3) / 4))
    .toString('base64') // convert to base64 format
    .slice(0, length) // return required number of characters
    .replace(/\+/g, '0') // replace '+' with '0'
    .replace(/\//g, '0') // replace '/' with '0'
}
```
You can [Look at this](https://blog.abelotech.com/posts/generate-random-values-nodejs-javascript/#generate-random-values-using-node-js-crypto-module) for more info.

Reply:
To generate a password we need a length of 20 characters